migreringer/src/main/resources/db/migration/defaultDS/4.1/V4.1_21__TFP-5688-5690-rydder_migrer_kilde_behandling_uuid_not_null.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE BEHANDLING SET UNUSED (MIGRERT_KILDE);
+ALTER TABLE BEHANDLING MODIFY UUID NOT NULL;
```

Comment:
Hm, 
query optimizer liker at det er not null - men MODIFY UUID NOT NULL tar tabellås og må innom alle radene.
Det er 1.4M rader - så den bør kjøre ganske fort når den får låsen siden det finnes en unik index -  men må kanskje vente på låsen pga aktivitet

Vet ikke om den vil ta lenger tid enn flyway-timeout - foreslår å ta den kommandoen  i et eget migreringsskript. 
Da er det enklere å rydde opp ved en timeout